### PR TITLE
Eclipse tidy

### PR DESCRIPTION
**Pull request by daveol**
format files using eclipse on request of @peternewman

### CODE REVIEW

File: olad/www/console_values.html [hunk 1]
```diff
@@ -1,728 +1,746 @@
 <html>
```
Comment (peternewman) [inline olad/www/console_values.html]: @nomis52 should this file even be in OLA, it seems to be a test one that snuck in ages ago.

**Comment by peternewman** [comment]
@nomis52 I don't know what your thoughts are on this generally; Eclipse seems to favour tabs in html, something we've generally avoided in the past. I guess there's also a question around how we manage autoformatting changes/which changes we take, to avoid a situation where we've just got two different bits of software with different ideas fighting against each other.

File: olad/OladHTTPServer.cpp [hunk 2]
```diff
@@ -1025,20 +961,19 @@ void OladHTTPServer::DecodePortIds(const string &port_ids,
     }
 
     unsigned int device_alias, port;
-    if (!StringToInt(tokens[0], &device_alias) ||
-        !StringToInt(tokens[2], &port)) {
+    if (!StringToInt(tokens[0], &device_alias)
+        || !StringToInt(tokens[2], &port)) {
```
Comment (nomis52) [inline olad/OladHTTPServer.cpp]: Was this generated by eclipse? We generally put the logic operator and the end of the line

File: olad/www/console_values.html (hunk 1 above)
Comment (nomis52) [inline olad/www/console_values.html]: I think this can be removed. I was used for testing years ago.

**Comment by nomis52** [comment]
I don't want to introduce tabs into the codebase. @daveol  can you configure Eclipse to use tabs instead of spaces please.

File: olad/OladHTTPServer.cpp [hunk 3]
```diff
@@ -894,14 +843,11 @@ void OladHTTPServer::HandleBoolResponse(HTTPResponse *response,
...
 
-
 /**
  * @brief Add the json representation of this port to the ostringstream
  */
-void OladHTTPServer::PortToJson(JsonObject *json,
-                                const OlaDevice &device,
-                                const OlaPort &port,
-                                bool is_output) {
+void OladHTTPServer::PortToJson(JsonObject *json, const OlaDevice &device,
```
Comment (nomis52) [inline olad/OladHTTPServer.cpp]: Is there a way to leave these one-per-line?

File: olad/OladHTTPServer.cpp [hunk 4]
```diff
@@ -154,15 +151,14 @@ bool OladHTTPServer::Init() {
     return false;
   }
   /*
-  Setup disconnect notifications.
-  m_socket->SetOnClose(
-    ola::NewSingleCallback(this, &SimpleClient::SocketClosed));
-  */
+   Setup disconnect notifications.
+   m_socket->SetOnClose(
+   ola::NewSingleCallback(this, &SimpleClient::SocketClosed));
```
Comment (peternewman) [inline olad/OladHTTPServer.cpp]: I think we want this line indented by two more.

File: olad/OladHTTPServer.cpp [hunk 5]
```diff
@@ -687,7 +647,7 @@ void OladHTTPServer::HandlePortsForUniverse(
     for (; iter != devices.end(); ++iter) {
       const vector<OlaInputPort> &input_ports = iter->InputPorts();
       for (input_iter = input_ports.begin(); input_iter != input_ports.end();
-           ++input_iter) {
+          ++input_iter) {
```
Comment (peternewman) [inline olad/OladHTTPServer.cpp]: This line wants reverting so it's indented by 1 more again.

File: olad/OladHTTPServer.cpp [hunk 6]
```diff
@@ -696,9 +656,8 @@ void OladHTTPServer::HandlePortsForUniverse(
 
       const vector<OlaOutputPort> &output_ports = iter->OutputPorts();
       for (output_iter = output_ports.begin();
-           output_iter != output_ports.end(); ++output_iter) {
-        if (output_iter->IsActive() &&
-            output_iter->Universe() == universe_id) {
+          output_iter != output_ports.end(); ++output_iter) {
```
Comment (peternewman) [inline olad/OladHTTPServer.cpp]: Again +1.

File: olad/OladHTTPServer.cpp [hunk 7]
```diff
@@ -737,14 +694,14 @@ void OladHTTPServer::HandleCandidatePorts(
   for (; iter != devices.end(); ++iter) {
     const vector<OlaInputPort> &input_ports = iter->InputPorts();
     for (input_iter = input_ports.begin(); input_iter != input_ports.end();
-         ++input_iter) {
+        ++input_iter) {
```
Comment (peternewman) [inline olad/OladHTTPServer.cpp]: Again.

File: olad/OladHTTPServer.cpp [hunk 8]
```diff
@@ -737,14 +694,14 @@ void OladHTTPServer::HandleCandidatePorts(
...
       JsonObject *obj = json.AppendObject();
       PortToJson(obj, *iter, *input_iter, false);
     }
 
     const vector<OlaOutputPort> &output_ports = iter->OutputPorts();
-    for (output_iter = output_ports.begin();
-         output_iter != output_ports.end(); ++output_iter) {
+    for (output_iter = output_ports.begin(); output_iter != output_ports.end();
+        ++output_iter) {
```
Comment (peternewman) [inline olad/OladHTTPServer.cpp]: Again.

**Pull request closed without merging**